examples/supabase/CommunityToolkit.Aspire.Supabase.Api/Program.cs

```diff
@@ -0,0 +1,62 @@
+//Using
+// - https://github.com/supabase-community/supabase-csharp/blob/master/Examples/SupabaseExample/Program.cs
+// - https://github.com/supabase-community/supabase-csharp/blob/master/Examples/BlazorWebAssemblySupabaseTemplate/Program.cs
+// as the example to setup supabase
+
+using CommunityToolkit.Aspire.Supabase.Api.Models;
+using Supabase.Postgrest.Responses;
+
+WebApplicationBuilder builder = WebApplication.CreateBuilder(args);
+
+string url = Environment.GetEnvironmentVariable("SUPABASE_URL") ?? "";
+string key = Environment.GetEnvironmentVariable("SUPABASE_KEY") ?? "";
```

Comment:
This should come from the .NET config API rather than environment variables.

Reply:
This was just for testing, i created a CommunityToolkit.Aspire.Supabase.Client project which will inject this into the supabase client.
I still need to see how to make the psql client integration